.github/workflows/release.yml

```diff
@@ -59,3 +59,13 @@ jobs:
       - name: Copy to latest
         if: env.pre_release == 'false'
         run: gsutil -m cp "gs://runme-artifacts-41eac6/${{ env.version }}/*" gs://runme-artifacts-41eac6/latest
+      - name: Bump Homebrew Formula
+        uses: mislav/bump-homebrew-formula-action@v2
+        # skip prereleases
+        if: ${{ !contains(github.ref, '-') }}
+        with:
+          formula-name: runme
+          homebrew-tap: Homebrew/homebrew-core
+          base-branch: master
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
According to the [docs for this action](https://github.com/mislav/bump-homebrew-formula-action):

> You should enable GITHUB_TOKEN only if the repository that runs this Action is private and if COMMITTER_TOKEN has the public_repo scope only. GITHUB_TOKEN will be used for verifying the SHA256 sum of the downloadable archive for this release.

Apparently we're supposed to create a new personal access token for this purpose, with the `repo` and `workflow` scopes.

Reply:
Do you think we can use the same token for both?

Reply:
@sourishkrout mind adding a repository secret? I don't have access to that section.

Reply:
You're admin on the repo now